src/providers/twitch/api/Helix.cpp

```diff
@@ -2704,6 +2704,7 @@ void Helix::updateShieldMode(
                     {
                         failureCallback(Error::UserMissingScope, message);
                     }
+                    [[fallthrough]];
```

Comment:
I don't think we mean to fallthrough here - this feels like a bug.
Should add a break here & add an else case the "Missing scope" check & call `failureCallback(Error::Forwarded, message);`